### PR TITLE
update for julia 0.5

### PR DESCRIPTION
**Pull request by CarloLucibello**
put a checkout for LG in travis just to check if everything is alright

### CODE REVIEW

**Comment by CarloLucibello** [comment]
I removed the combinatorial_adjacency function and GraphMatrices dependence, since they will be part of LG
  
this should  ready to merge

**Comment by sbromberger** [comment]
We need to wait until https://github.com/JuliaGraphs/LightGraphs.jl/pull/472 and https://github.com/JuliaGraphs/LightGraphs.jl/pull/473 are merged before we do this - otherwise I think we're going to lose functionality in the interim.

**Comment by CarloLucibello** [comment]
well, right now 0.5 users have no functionality at all, they can't install the package...

File: REQUIRE [hunk 1]
```diff
@@ -1,7 +1,6 @@
-julia 0.4 0.5-
-LightGraphs
+julia 0.5-
+LightGraphs 0.7.1
```
Comment (sbromberger) [inline REQUIRE]: Can you bump this to 0.7.2 please?

**Comment by sbromberger** [comment]
@CarloLucibello I'm ready to ignore the nightly failure if you think this is ready to go.

**Comment by CarloLucibello** [comment]
go!

**Pull request merged**